### PR TITLE
Fixes the bug of sending an e-mail notification only to the owner of the...

### PR DESCRIPTION
**Pull request by nicolashainaux**
... event. Also allows to send e-mail notifications to the members of groups that are shared with. Also checks that each recipient gets the e-mail only once, even if it belongs to several groups, for instance.

### CODE REVIEW

**Comment by georgehrke** [comment]
Code looks good, will test asap

@quamenzullo Thx for the pull request :)

File: lib/app.php [hunk 1]
```diff
@@ -551,6 +551,18 @@ public static function sendEmails($eventid, $summary, $location, $description, $
 			}
 			foreach ($eventsharees as $sharee) {
 				$eventShareesNames[] = $sharee['share_with'];
+				$shwth = $sharee['share_with'];
+                                if(OC_Group::groupExists($shwth)) {
```
Comment (georgehrke) [inline lib/app.php]: users and groups can have the same id, please check `$sharee['share_type']` instead

**Comment by nicolashainaux** [comment]
I have too much work these days but I'll check soon what you've said here

File: lib/app.php (hunk 1 above)
Comment (nicolashainaux) [inline lib/app.php]: OK, I've checked it, corrected it on my server too, it works fine, and in addition I noticed I made an error in this commit (the line right above, 553, should not be there). I will create a new commit correcting this and pull a new request. This one can be removed.

**Comment by nicolashainaux** [comment]
OK, I think I don't master the github way of managing pull requests, I added a new commit which corrects the errors. It seems it has been added to the pull request, though I am not completely sure. Tell me if I must do something else.

**Comment by georgehrke** [comment]
Code looks good now, will test this asap. Thx for the pull request!

**Pull request merged**